### DISCUSSION TITLE
Welcome to albumentations Discussions!

### DISCUSSION THREAD

**Discussion by creafz** [albumentations-team/albumentations#779] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ranjith-12345** [comment]
I am trying to use albumentations.augmentations.domain_adaptation from google collab. However, I am getting the following error:

ModuleNotFoundError: No module named 'albumentations.augmentations.domain_adaptation

**Comment by v-bliznyukov** [comment]
Hello! Is it possible to set a random seed, so that results are reproducible? Thank you!

**Reply by Dipet** [reply]
Try this commands:
```
import random
import numpy as np

random.seed(0)
np.random.seed(0)
```
